tldraw/tldraw
# Add fit to content for frames.

Adds Fit to content option for frames. This resizes the frames so that the whole content fits. It also adds 50px padding on all sides so that the content does not touch the frame's borders.


https://github.com/tldraw/tldraw/assets/2523721/b2f86e31-7dfb-495f-ac31-f1e0125e0af1


https://github.com/tldraw/tldraw/assets/2523721/e0a73d25-ac9f-4a35-a1fd-4aed7a5b151c



Fixes #1407

### Change Type


[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Test Plan

1. Add some shapes.
2. Add a frame that encloses those shapes.
3. Right click on the frame and choose `Fit to content`
4. The frame should resize to fit all the children with some padding on all sides of the frame.


### Release Notes

- Add Fit to content option to the context menu for frames. This resizes the frames to correctly fit all their content.
